docs/source/how_to_add_a_component.rst

```diff
@@ -0,0 +1,112 @@
+How to Add a Component
+======================
+
+Intro
+-----
+
+To integrate a new component into the dashboard you need to add a folder (e.g.
+`maps`) in `utilities/dashboard/components`. The code that generates your component
+should be divided into three modules:
+
+- plotting module (e.g. `mapplot.py`), containing functions that process
+  a relatively "raw" dataset and returns the statistics to be displayed in the
+  plot, and functions that generate the component's plot.
```

Comment:
Maybe explain here also what is **not** in the plotting module. Although, this would probably become more self-apparent if this was not the first module you described.